contracts/seed/Seed.sol

```diff
@@ -133,6 +133,19 @@ contract Seed {
         _;
     }
 
+    modifier isClosed() {
+        require(!closed, "Seed: should not be closed");
```

Comment:
```suggestion
    modifier isNotClosed() {
        require(!closed, "Seed: should not be closed");
```

Either modifier name or require should change.
As I read it, it's the direct opposite.

Reply:
Then I would say the require need to change because I tried to keep the modifer naming in line with the others, i.e. use `is` instead of `isNot`

Reply:
Changed